api/graphql/schema.graphql

```diff
@@ -866,7 +866,11 @@ type CandidateSearchPoolResult {
 type Query {
   me: User @auth
   user(id: UUID! @eq): User @find @guard @can(ability: "view", query: true)
-  users: [User]! @all @guard @can(ability: "viewAny")
+  users: [User]!
+    @all
+    @guard
+    @can(ability: "viewAny")
+    @deprecated(reason: "Removing in favour of usersPaginated query")
```

Comment:
I know that we don't have a standard for writing the reason, but what better time to start one than now!

Suggestion: "_oldField_ is deprecated. Use _newField_ instead."

Reply:
Does this look good to you? [update messages](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7641/commits/306553ce1c359b224dd2dd322aa44165725eab05)